vscode/src/client/analyzerClient.ts

```diff
@@ -50,34 +51,69 @@ export class AnalyzerClient {
       mutateExtensionState((draft) => {
         draft.isFetchingSolution = flag;
       });
-    this.outputChannel = vscode.window.createOutputChannel("Konveyor-Analyzer");
+
+    this.outputChannel = vscode.window.createOutputChannel("Konveyor-Analyzer"); //, { log: true });
+
+    this.assetPaths = buildAssetPaths(extContext);
     this.kaiDir = path.join(buildDataFolderPath()!, "kai");
     this.kaiConfigToml = path.join(this.kaiDir, "kai-config.toml");
+
+    this.outputChannel.appendLine(
+      `current asset paths: ${JSON.stringify(this.assetPaths, null, 2)}`,
+    );
+    this.outputChannel.appendLine(`Kai directory: ${this.kaiDir}`);
+    this.outputChannel.appendLine(`Kai config toml: ${this.kaiConfigToml}`);
   }
 
   public async start(): Promise<void> {
     if (!this.canAnalyze()) {
-      vscode.window.showErrorMessage("Cannot start the server due to missing configuration.");
+      vscode.window.showErrorMessage(
```

Comment:
The lint-staged process should apply everything all nice and neat if I don't fix it before committing...

But also in this case, I changed the text to read "the kai rpc server" instead of just "the server" to try to be consistent everywhere.